app/_hub/_init/my-extension/index.md

```diff
@@ -136,8 +136,6 @@ kong_version_compatibility: # required
 
 params: # metadata about your plugin
   name: # name of the plugin in Kong (may differ from name: above)
-  api_id:
```

Comment:
We can't remove this - it is needed for plugin versions that still support APIs.